FND/GoalsAndObjectives/Objectives.rdf

```diff
@@ -109,6 +110,17 @@
 		<skos:definition>short-term, quantitative, measurable result that a person, organization, or system seeks to attain in order to achieve its long-term goals</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-gao-obj;Program">
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-gao-obj;hasObjective"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-gao-obj;Objective"/>
```

Comment:
not sure about this, assuming that an objective is measurable and time-based. A Project has objectives, a Program(me) is larger in scope, more open-ended and encompasses many projects and has goals rather than objectives. Here is a definitive link https://www.prince2.com/usa/blog/project-vs-programme from the official project management method in the UK

Reply:
I used the BLS definition while working with Dan when you were not on the call to do this, but I do like some of the things outlined on the site you've referenced, and will add some of them.   Note that I think it makes sense to merge the Goals ontology with this Objectives ontology and then reference the goal, since there is only one tiny class in the Goals ontology and it's only referenced a couple of times (and there was no longer a hasGoal property that I found, though there was a change note saying we removed it at some point).

Reply:
I agree with removing the separate Goals ontology. Even the Objectives one is both limited and unnecessarily specific (e.g. subclasses of "Strategy"). If serious about this area we could look at BMM and/or StratML.